json-rpc/tests/integration_test.rs

```diff
@@ -494,6 +494,10 @@ fn create_test_cases() -> Vec<Test> {
                             "gas_unit_price": 0,
                             "max_gas_amount": 1000000,
                             "public_key": sender.public_key.to_string(),
+                            "secondary_signers": [],
+                            "secondary_signature_schemes": [],
+                            "secondary_signatures": [],
+                            "secondary_public_keys": [],
```

Comment:
could we add a test that actually has secondary signers and signatures?

Reply:
Added one that uses the current admin script.